src/nifs.rs

```diff
@@ -162,8 +162,11 @@ mod tests {
     Ok(())
   }
 
-  #[test]
-  fn test_tiny_r1cs_bellperson() {
+  fn test_tiny_r1cs_bellperson_with<S, G>()
+  where
+    S: PrimeField,
```

Comment:
@huitseeker A minor nit. Is it necessary for this to be generic over S, when it is already generic over G?

Reply:
No, you're right, here and below, I was re-using the defined alias for `S`, but that's a bit lazy. Will fix.